src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const CAR_PRICE = 40;
+  const SHORT_PERIOD = 3;
+  const SHORT_PERIOD_DISCOUNT = 20;
+  const LONG_PERIOD = 7;
+  const LONG_PERIOD_DISCOUNT = 50;
+
+  if (days >= LONG_PERIOD) {
+    return days * CAR_PRICE - LONG_PERIOD_DISCOUNT;
```

Comment:
move `days * CAR_PRICE` to a variable (Example: `baseRent`)
```suggestion
    return baseRent - LONG_PERIOD_DISCOUNT;
```